app/views/incomes/YourIncomeCalculationView.scala.html

```diff
@@ -32,155 +32,146 @@
     ec: scala.concurrent.ExecutionContext)
 
 
-@printLink= {
-    <p class="flush--bottom">
-        @includes.link(
-            url=routes.YourIncomeCalculationController.printYourIncomeCalculationPage(model.empId).toString,
```

Comment:
Is it worth raising another ticket to use the new way of printing rather than going to a separate URL?

Reply:
I think Udnan has some tickets around this ^